test/utils/SchemaTester.js

```diff
@@ -0,0 +1,108 @@
+const testSchema = (Model) => ({
+    fieldRequired: (fieldName, customObject) => {
+        test(`'${fieldName}' is required`, async () => {
+            const emptyObject = new Model(customObject || {});
```

Comment:
Given that it might not be empty, it makes more sense for `emptyObject` to be renamed to `modelInstance` imo